CS3230Project/View/Checkup.cs

```diff
@@ -301,5 +336,35 @@ private void NotCompletedTestsTable_CellContentClick(object sender, DataGridView
         {
             SwitchForms.Switch(this, new AddTestResults(this.testManager.NotPerformedTests[e.RowIndex], this.appointmentId, this.patient, this.doctor));
         }
+
+        private void SubmitDiagnosis_Click(object sender, EventArgs e)
```

Comment:
When I submit a diagnosis with empty results the program crashes. Validation needs to be added in the case of a diagnosis with empty results.

Reply:
I completely forgot to add the validation. I will get that added tonight.

Reply:
Fixed.